app/controllers/group_order_invoices_controller.rb

```diff
@@ -0,0 +1,38 @@
+class GroupOrderInvoicesController < ApplicationController
+  include Concerns::SendGroupOrderInvoicePdf
+
+  def show
+    @group_order_invoice = GroupOrderInvoice.find(params[:id])
+    if FoodsoftConfig[:contact][:tax_number]
+      respond_to do |format|
+        format.pdf do
+          send_group_order_invoice_pdf @group_order_invoice if FoodsoftConfig[:contact][:tax_number]
```

Comment:
you check already for the `tax_number` in line 6. what's the purpose of the second check here?